### PR TITLE
Update measurement tools with internal updates

### PR DESCRIPTION
**Pull request by calebmshafer**
Pull over the internal updates made to the tool as not everyone is using this version yet.

After this change, we can hopefully make that the case and all new updates will be made directly to this package.

### CODE REVIEW

File: packages/measure-tools/src/api/FeatureTracking.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
+* See COPYRIGHT.md in the repository root for full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
 import { BeUiEvent, GuidString, Logger } from "@bentley/bentleyjs-core";
```
Comment (aruniverse) [inline packages/measure-tools/src/api/FeatureTracking.ts:6]: hate that this exists, we should chat about this later

File: packages/measure-tools/src/tools/MeasureAreaTool.ts [hunk 2]
```diff
@@ -102,7 +102,7 @@ export class MeasureAreaTool extends MeasurementToolBase<AreaMeasurement, Measur
           hints.setRotation(mat.inverse()!);
       }
     }
-    hints.sendHints(true);
+    hints.sendHints(false);
```
Comment (aruniverse) [inline packages/measure-tools/src/tools/MeasureAreaTool.ts:105]: what are the points of hints if we dont show them? 
@a-gagnon @bsy-nicholasw

**Review by aruniverse — APPROVED** [review]

File: packages/measure-tools/src/api/FeatureTracking.ts (hunk 1 above)
Comment (calebmshafer) [reply packages/measure-tools/src/api/FeatureTracking.ts:6]: lol yea, I'll clean all this up in a separate PR.

File: packages/measure-tools/src/tools/MeasureAreaTool.ts (hunk 2 above)
Comment (aruniverse) [reply packages/measure-tools/src/tools/MeasureAreaTool.ts:105]: spoke with nicholas about this, users need to manually turn on accusnap to see these now

**Pull request merged**